Mathlib/GroupTheory/Perm/Support.lean

```diff
@@ -228,10 +228,8 @@ section Set
 
 variable (p q : Perm α)
 
-theorem set_support_inv_eq : { x | p⁻¹ x ≠ x } = { x | p x ≠ x } := by
-  ext x
-  simp only [Set.mem_setOf_eq, Ne]
-  rw [inv_def, symm_apply_eq, eq_comm]
+lemma set_support_inv_eq : {x | p⁻¹ x ≠ x} = {x | p x ≠ x} := by
```

Comment:
Any particular reason for changing from `theorem` to `lemma`?

Reply:
Whoops, that's accidental shuffle